src/Components/image/Image.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Text from "../../elements/text/Text";
+import styles from "./Image.module.scss";
+import PropTypes from "prop-types";
+
+const Image = ({ image, children, desc }) => {
```

Comment:
Let's rename this to CardImage and leave Image component for actual image. 
We will need that in the future.

Reply:
Noted